ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TutorialOnboardingLayout.kt

```diff
@@ -0,0 +1,316 @@
+@file:Suppress("UNUSED_EXPRESSION")
+
+package org.cru.godtools.tutorial.layout
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.LottieConstants
+import com.airbnb.lottie.compose.animateLottieCompositionAsState
+import com.airbnb.lottie.compose.rememberLottieComposition
+import org.ccci.gto.android.common.androidx.compose.foundation.text.minLinesHeight
+import org.cru.godtools.base.ui.theme.GodToolsTheme
+import org.cru.godtools.tutorial.Page
+import org.cru.godtools.tutorial.R
+
+@Composable
+internal fun TutorialOnBoardingWelcomeLayout(
+    nextPage: () -> Unit = {},
+    onTutorialAction: (Int) -> Unit = {},
+) = GodToolsTheme() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(
+                top = dimensionResource(R.dimen.tutorial_page_inset_top),
+                bottom = dimensionResource(R.dimen.tutorial_page_inset_bottom)
+            )
+            .padding(horizontal = 32.dp)
+            .verticalScroll(rememberScrollState())
+    ) {
+        Spacer(modifier = Modifier.weight(1f))
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Column() {
+                Spacer(modifier = Modifier.minLinesHeight(1, MaterialTheme.typography.titleLarge))
+                Spacer(modifier = Modifier.padding(top = 16.dp).minLinesHeight(5, MaterialTheme.typography.bodyMedium))
+            }
```

Comment:
There should be no need for wrapping the main content in a box with hidden spacers on the welcome page.

Reply:
Which will allow you to flatten out the welcome page into a single column